cluster-provision/k8s/1.30/k8s_provision.sh

```diff
@@ -62,15 +64,25 @@ function pull_container_retry() {
     fi
 }
 
-export CRIO_VERSION=1.30
-cat << EOF >/etc/yum.repos.d/devel_kubic_libcontainers_stable_cri-o_${CRIO_VERSION}.repo
+if [ "$arch" == "s390x" ]; then
+  export CRIO_VERSION=1.28
+  #As crio version available in kubevirtci-crio-mirror/isv_kubernetes_addons_cri-o_stable_v1.30 is broken with issue https://github.com/containers/crun/issues/1494, which is fixed, but yet to part of crio release
```

Comment:
Is this still true? I saw there were recent crio releases?

Reply:
Not sure if still true, I've tried a month back. So, let me once try with latest available crio [v1.30.5](https://github.com/cri-o/cri-o/releases/tag/v1.30.5) (or probably even I'll with v1.31.0 if 1.30 doesn't work)

Reply:
reverted